app/assets/stylesheets/components/_buttons.scss

```diff
@@ -16,6 +16,22 @@
 // TIPS FOR PRIMARY RED BUTTON You can include a primary red button like a "PARTICIPATE" button in HTML as following:
 // <%= link_to "PARTICIPATE", event_path(@event) , class: "btn-red" %>
 
+.btn-red-create {
```

Comment:
It'd be better to create an #id for the specific overwrites you need for this specific button or create a .class that enables for some group of buttons.